addons/website_blog/controllers/main.py

```diff
@@ -1,7 +1,11 @@
 # -*- coding: utf-8 -*-
 
+from feedgen.feed import FeedGenerator
```

Comment:
@antonylesuisse Can we add a new dependency to this lib? https://pypi.python.org/pypi/feedgen/0.3.1